.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "frame/ddc-pallet"]
```

Comment:
Switch to the `dev` branch after DDC-pallet PR will be merged.

Reply:
This specify the target path. I will not change this. But i will add parameter to specify the dev branch in the next commit after you merge the PR in ddc-pallet repo ([url](https://github.com/Cerebellum-Network/ddc-pallet/pull/1))